src/shared/html-content/html-content.ts

```diff
@@ -1,25 +1,27 @@
 import { css } from "styled-components";
 import { Color } from "../../color";
+import { FontFamily } from "../../spec/text-spec/font-spec";
 import { TextLinkSizeType, TextSizeType, TextStyleHelper } from "../../text";
 
-interface HtmlContentStyleOptions {
-    textColor?: string | ((props: any) => string);
+export interface HtmlContentStyleOptions {
+    textSize?: TextSizeType | TextLinkSizeType | undefined;
+    textColor?: string | ((props: any) => string) | undefined;
```

Comment:
why the need to pass textColor? technically we can style the color after `applyHtmlContentStyle()` like

```
const MyComponent = styled.div`
   ${applyHtmlContentStyle({ textSize: "Body"});
   color: Color.Neutral[1];
}
`;
```

similarly for `Markup`, if we want to change the colour, we will turn it into a styled-component

anyway if this is intended, we can introduce generics to the textColor function to allow devs to narrow the arguments `(<T = any>(props: T) => string)`

Reply:
agreed, it's not necessary for `applyHtmlContentStyle` now that it can inherit the text color

for the `Markup` component it's more for convenience

Reply:
> we can introduce generics to the textColor function to allow devs to narrow the arguments

the function is for theme colors, so the idea is we can specify `Color.Primary` etc. and the props type doesn't matter